assets/messages.yml

```diff
@@ -108,6 +108,25 @@ messages:
         *Thank you for your report!*
         Could you please include information about your server? Without it we may be unable to reproduce the issue.
 
+        %quick_links%
+      fillname: []
+  attach-video:
+    - project:
+        - mc
+        - mcl
+        - mcpe
+        - mce
```

Comment:
Maybe there have to be two separate messages, one for mobile devices (MCPE, MCE) and one for desktop devices (MC, MCL, MCD), becase the suggestion for using OBS will not work for mobile devices. Or that suggestion should be clarified and another suggestion for recording on mobile devices should be added (in case there are good, ad-free, free apps for recording).

Reply:
Kept MCPE for both, as there is a Windows 10 version of the Bedrock game.